src/olympia/access/migrations/0003_add_force_send_mail_group.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+from olympia.access.models import Group
```

Comment:
Never import an actual django model in a migration: you would get the actual model as currently defined instead of the proxy for the historical state it's supposed to be at the time of the migration, causing potential failures.

This is why the callbacks receive an `apps` argument, you want to do (inside the functions themselves):
```python
Group = apps.get_model('access', 'Group')
```

Reply:
whoops! missed that!

Reply:
Done.